tests/sentry/tasks/test_weekly_escalating_forecast.py

```diff
@@ -4,12 +4,14 @@
 from unittest.mock import patch
 
 from sentry.issues.escalating import GroupsCountResponse
+from sentry.issues.escalating_group_forecast import EscalatingGroupForecast
 from sentry.models.group import Group, GroupStatus, GroupSubStatus
-from sentry.models.groupforecast import GroupForecast
 from sentry.models.project import Project
 from sentry.tasks.weekly_escalating_forecast import run_escalating_forecast
 from sentry.testutils.cases import APITestCase, SnubaTestCase
 
+FORECAST_LIST_MOCK = [200] * 14
```

Comment:
```suggestion
DEFAULT_MINIMUM_CEILING_FORECAST = [200] * 14
```

Maybe this?

Reply:
This is a good idea, thank you for the suggestion